.github/workflows/maestro_release.yaml

```diff
@@ -37,8 +39,16 @@ jobs:
           python-version: "3.12"
       - name: Install uv
         uses: astral-sh/setup-uv@v6
+      - name: Install dependencies
+        run: |
+          uv pip install -e .[crewai]
```

Comment:
this shouldn't be needed, the python and uv install steps should be enough to run the py script (if it's not then it should just move to the test step)